lms/services/course.py

```diff
@@ -6,7 +6,7 @@
 
 class CourseService:
     def __init__(self, application_instance_service, consumer_key, db):
-        self._application_instance_service = application_instance_service
+        self._application_instance = application_instance_service.get()
```

Comment:
I think this is a better pattern, if there's no AI something is very wrong, better find out soon. We are not handling ConsumerKeyError (but neither were we before). We might need to handle that globally if it ever becomes a problem.